### ISSUE TITLE
cpptraj hangs forever with bad prmtop

### ISSUE THREAD

**Issue by hainm** [Amber-MD/cpptraj#389]
FYI: `cpptraj.OMP -p $AMBERHOME/test/dna_pol/prmtop`

(Ideally it should exit).

```bash
$ ./cpptraj -p $AMBERHOME/test/dna_pol/prmtop

CPPTRAJ: Trajectory Analysis. V17.00b
    ___  ___  ___  ___
     | \/ | \/ | \/ | 
    _|_/\_|_/\_|_/\_|_

| Date/time: 08/11/16 17:08:09
| Available memory: 757.809 MB

	Reading '/home/haichit/amber_git/amber/test/dna_pol/prmtop' as Amber Topology
	Reading old (<v7) Amber Topology file.
Warning: Box: Unrecognized beta (3); setting all angles to beta.
Warning: Non-orthogonal box is too skewed to perform accurate imaging.
Warning:  Images and imaged distances may not be the absolute minimum.
	This Amber topology does not include atomic numbers.
	Assigning elements from atom masses/names.
Warning: Could not determine atomic number from mass (70.000000) [22  ]
Warning: Could not determine atomic number from mass (1000000.000000) [
N  ]
Warning: Could not determine atomic number from mass (800000.000000) [4  H]
Warning: Could not determine atomic number from mass (800000.000000) [41 H]
Warning: Could not determine atomic number from mass (1000000.000000) [42 N]
Warning: Could not determine atomic number from mass (600000000.000000) [2  O]
Warning: Could not determine atomic number from mass (100800000.000000) [3' H]
Warning: Could not determine atomic number from mass (1201000000.000000) [3' C]
Warning: Could not determine atomic number from mass (100800000.000000) [2' H]
Warning: Could not determine atomic number from mass (100800000.000000) [2'1H]
Warning: Could not determine atomic number from mass (1600000000.000000) [2'2O]
Warning: Could not determine atomic number from mass (3.000000) [3' L]
Warning: Could not determine atomic number from mass (16.000000) [1P O]
Warning: Could not determine atomic number from mass (16.000000) [2P 
]
	No SCEE section: setting Amber default (1.2)
	No SCNB section: setting Amber default (2.0)
Error: Atom 7 was assigned a lower molecule # than previous atom.
Error:   This can happen if bond information is incorrect or missing, or if the
Error:   atom numbering in molecules is not sequential. Try one of the
Error:   following:
Error: - If this is a PDB file, try using the 'noconect' keyword.
Error: - If this topology did not have bond info, try increasing the bond
Error:   search cutoff above 0.2 Ang. ('bondsearch <cutoff>').
Error: - Use the 'fixatomorder' command to reorder the topology and any
Error:   associated coordinates.
Error: - Use the 'setMolecules' command in parmed to reorder only the
Error:   topology.
Error: Could not determine molecule information for prmtop.
Error: SetSolventInfo: No molecule information.
Error: Could not determine solvent information for prmtop.

```

**Comment by hainm** [comment]
PS: if using `pytraj` for this prmtop, I got segmentation fault. That's why I raise this issue.

**Comment by drroe** [comment]
Should be fixed by #390.

**Comment by hainm** [comment]
confirmed. thanks.

```bash
In [4]: pt.load_topology('/home/haichit/amber_git/amber16/test/dna_pol/prmtop')
Out[4]: <Topology: 12423 atoms, 3851 residues, 3829 mols, PBC with box type = ortho>
```
